test/pubsub.jl

```diff
@@ -2,7 +2,7 @@
 #works alongside echonode.py
 #typegeneration.jl must be run first
 
-using geometry_msgs.msg
+using .geometry_msgs.msg
```

Comment:
See https://github.com/JuliaLang/julia/issues/17997 and https://github.com/JuliaLang/julia/pull/23579.